modules/extensions-soap-support/src/main/java/org/mule/runtime/module/extension/soap/internal/loader/InvokeOperationDeclarer.java

```diff
@@ -59,7 +59,7 @@ public OperationDeclarer declare(ExtensionDeclarer declarer, ClassTypeLoader loa
     OperationDeclarer operation = declarer.withOperation(OPERATION_NAME).describedAs(OPERATION_DESCRIPTION)
         .withModelProperty(new OperationExecutorModelProperty(new SoapOperationExecutorFactory()));
 
-    operation.withOutput().ofType(TYPE_BUILDER.stringType().build());
+    operation.withOutput().ofType(TYPE_BUILDER.anyType().build());
```

Comment:
this isn't good either. Any would give a very bad experience to the user.

1) if only strings can go out, leve it as it was but add a todo to support streaming
2) if only InputStream is allowed, then change to binary
3) It it can either be a String or a Multipart, then return a union type.